release/serve_tests/workloads/serve_test_utils.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import re
+
+
+def parse_wrk_decoded_stdout(decoded_out):
+    """
+    Parse decoded wrk stdout to a dictionary.
+
+    # Sample wrk stdout:
+    #
+    # Running 10s test @ http://127.0.0.1:8000/echo
+    # 2 threads and 84 connections
+    # Thread Stats   Avg      Stdev     Max   +/- Stdev
+    #     Latency    59.33ms   13.51ms 113.83ms   64.20%
+    #     Req/Sec   709.16     61.73   848.00     78.50%
+    # 14133 requests in 10.02s, 2.08MB read
+    # Requests/sec:   1410.71
+    # Transfer/sec:    212.16KB
+
+    Returns:
+        metrics_dict (Dict[str, str]):
+            {'latency_avg': '66.57ms', 'latency_stdev': '4.47ms',
+             'latency_max': '113.34ms', 'latency_+/-_stdev': '91.33%',
+             'req/sec_avg': '180.88', 'req/sec_stdev': '26.83',
+             'req/sec_max': '323.00', 'req/sec_+/-_stdev': '65.22%',
+             'requests/sec': '1440.06', 'transfer/sec': '216.57KB'}
```

Comment:
can you make the value number so they are be plotted on a line chart? you can move the unit to key if you want, like `latency_avg_ms`